beetsplug/replaygain.py

```diff
@@ -436,29 +462,36 @@ def _construct_cmd(self, item, peak_method):
             "-",
         ]
 
-    def _analyse_item(self, item, count_blocks=True):
+    def _analyse_item(self, item, target_level, peak, count_blocks=True):
         """Analyse item. Return a pair of a Gain object and the number
         of gating blocks above the threshold.
 
         If `count_blocks` is False, the number of gating blocks returned
         will be 0.
         """
+        target_level_lufs = db_to_lufs(target_level)
+        peak_method = {
+            Peak.none: "none",
+            Peak.true: "true",
+            Peak.sample: "sample",
+        }[peak]
+
         # call ffmpeg
         self._log.debug(u"analyzing {0}".format(item))
-        cmd = self._construct_cmd(item, self._peak_method)
+        cmd = self._construct_cmd(item, peak_method)
         self._log.debug(
             u'executing {0}', u' '.join(map(displayable_path, cmd))
         )
         output = call(cmd).stderr.splitlines()
 
         # parse output
 
-        if self._peak_method == "none":
+        if peak is Peak.none:
```

Comment:
The [Python docs](https://docs.python.org/3/library/enum.html) suggest that enum values should be compared with equality (`==`), not identity (`is`).